### DISCUSSION TITLE
不知道有没有人把这个插件和【浮图秀】一起用的

### DISCUSSION THREAD

**Discussion by ywbear** [the1812/Bilibili-Evolved#4963] [综合讨论 / General]
前两天【浮图秀】更新之后，好像和本插件的【自定义顶栏】功能有冲突，我排查了好一会应该是浮图秀的问题

**Comment by Geeyun-JY3** [comment]
之前不少人报过侧边栏的问题：<https://github.com/the1812/Bilibili-Evolved/issues?q=%E6%B5%AE%E5%9B%BE%E7%A7%80>，如 #2814、#3115

**Comment by magicdawn** [comment]
确实浮图秀只禁用功能还是触发变大, 必须禁用整个插件
